src/components/GWL.vue

```diff
@@ -554,12 +554,12 @@ export default {
           .end()
           .then(() => this.animateLine(self.current_time))
 
-          this.d3.selectAll(".ticker-date")
-          .transition()
-            .duration(this.day_length) 
-            .text(this.dates[start])
-          .end()
-          .then(() => this.animateLine(start))
+          // this.d3.selectAll(".ticker-date")
+          // .transition()
+          //   .duration(this.day_length) 
+          //   .text(this.dates[start])
+          // .end()
+          // .then(() => this.animateLine(start))
```

Comment:
Was this not getting used?

Reply:
Yeah, the code that [adds the date ticker](https://github.com/USGS-VIZLAB/gw-conditions/blob/main/src/components/GWL.vue#L599-L608) was already commented out, as was the [end animation for it](https://github.com/USGS-VIZLAB/gw-conditions/blob/main/src/components/GWL.vue#L574-L578), so I commented out this piece since it's not currently being used.